packages/server/modules/core/graph/resolvers/projects.ts

```diff
@@ -284,6 +284,13 @@ export = {
         throw new RateLimitError(rateLimitResult)
       }
 
+      const canCreate = await context.authPolicies.project.canCreateLegacy({
+        userId: context.userId
+      })
+      if (canCreate.isErr) {
+        throw canCreate.error
```

Comment:
Just an example of how the switch mapping could be avoided and auth errors could be thrown as is